index.js

```diff
@@ -288,6 +292,12 @@ BFFS.prototype.publish = function publish(spec, options, fn) {
     }
 
     if (error) return fn(error);
+
+    if (!error) {
+      this.log('Published build for %s', spec.name, spec,
```

Comment:
it will get replaced for `spec.name`. the rest of the arguments need something to handle the replacing.